### ISSUE TITLE
Get a `gh-pages` branch.

### ISSUE THREAD

**Issue by deavmi** [nickdesaulniers/What-Open-Source-Means-To-Me#79]
It would very cool if we could automate a webiste for the `README.md` file so that we could get it on the web like that. Just to make it look even more professional.

**Comment by nickdesaulniers** [comment]
Yeah, might need something like Jekyll for converting markdown to HTML though.

**Comment by justinclift** [comment]
@nickdesaulniers Nah, you're overthinking it.  GitHub projects can have an generated website with just a few clicks.  It's what we do for sqlitebrowser.org, which is really this:

&nbsp; &nbsp; https://github.com/sqlitebrowser/sqlitebrowser/tree/gh-pages

That was originally generated using "GitHub Pages" for the repo, which you'll find in:

&nbsp; &nbsp; https://github.com/nickdesaulniers/What-Open-Source-Means-To-Me/settings

We've since been manually adjusting ours, but it's completely possible to just keep regenerating it using GitHub Pages when the README.md changes.  It may be possible to automate with a Git trigger or something to... I haven't checked though. :wink:

**Comment by deavmi** [comment]
I just got one up here:D :smile: http://deavmi.github.io/What-Open-Source-Means-To-Me

**Comment by nickdesaulniers** [comment]
Up and running: https://nickdesaulniers.github.io/What-Open-Source-Means-To-Me/ .  @justinclift, you should add you name at the bottom next to mine.  We also need to fix up the images.

**Comment by justinclift** [comment]
I think it gets the name from the repo owner.  If I manually adjust it, then the next time the page is regenerated (eg after the next entry is added), I think the adjustment will be lost.

So, you're stuck being the only official owner for now. :grin:

**Comment by reqshark** [comment]
have you guys thought about using submarine? 

https://github.com/muan/submarine

i think it's really cool plus you could augment with your own styles, but the best I think, would be a random generator so it lands on a random quote and then you can cycle through them from there

![screen shot 2015-03-02 at 1 32 46 pm](https://cloud.githubusercontent.com/assets/937826/6447495/dd525e62-c0e0-11e4-824c-6b610df55127.png)

**Comment by justinclift** [comment]
@reqshark Is it something that works through GitHub Pages?  Looking at it quickly, it doesn't seem like it.

**Comment by nickdesaulniers** [comment]
eh, we need a precommit hook to keep this branch up to date with master
